dapps/tests/app/config/contracts.js

```diff
@@ -40,10 +40,10 @@ module.exports = {
         args: [100]
       },
       ContractArgs: {
-        args: [
-          ["$MyToken2", "$SimpleStorage"],
-          123
-        ]
+        args: {
+          initialValue: 123,
+          _addresses: ["$MyToken2", "$SimpleStorage"]
+        }
```

Comment:
Prior to this commit, this change will make tests fail. So we consider this an added test for the fix.

Reply:
It would be better to add it to an actual test file, since the contract config is only run when we manually do `embark run` on them